### PR TITLE
Update to django.po

### PR DESCRIPTION
**Pull request by auzelac91**
Translation in progress.

### CODE REVIEW

File: sr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -118,10 +122,10 @@ msgid ""
...
 "looking for, exactly when they need it."
 msgstr ""
-"Firefox OS se prilagođava potrebama kupaca bolje od bilo kog drugog "
-"telefona. Prilagodljiva pretraga aplikacija pruža informacije o svakoj temi "
-"i obezbeđuje VVašim kupcima više podataka koje traže, baš u trenutku kada su"
-" im oni potrebni. To im omogućuje da maksimalno iskoriste svaki trenutak."
+"Firefox OS se prilagođava potrebama potrošača bolje od bilo kog drugog "
+"telefona, predviđajući njihove potrebe i u trenu pružajući sve informacije "
+koje žele. Sa prilagodljivom pretragom aplikacija, Vaši potrošači će dobiti "
+više od onoga što traže, tačno kada im je potrebno."
```
Comment (jgmize) [inline sr/LC_MESSAGES/django.po]: These lines are missing the starting `"`

File: sr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -372,7 +380,7 @@ msgstr ""
 #. Page path: /1-4/demo-tips/
 #: db-strings.txt:438 db-strings.txt.py:7098
 msgid "SijuLONPI9U"
-msgstr "Wur2jVusiDI"
+msgstr "SijuLONPI9U"
```
Comment (jgmize) [inline sr/LC_MESSAGES/django.po:383]: Apologies that this is unclear, but these random looking strings are actually youtube ids and should not be translated to match. I'll look into automatically adding a comment to that effect to reduce the confusion caused by this.

**Comment by auzelac91** [comment]
Thanks, I'll correct the mistakes at line 127 and 128 once the file is merged.
One question about the youtube links... how can I know which youtube id I need to put for my locale?

**Comment by jgmize** [comment]
@auzelac91 sorry again about the confusion on the youtube ids. The intention was for you to ignore them, but we didn't think to add a comment to that effect in the file. I'm working on an alternative solution that will remove them from the po files entirely now.

**Comment by jgmize** [comment]
@auzelac91 we can't merge this as-is because the syntax errors (the missing `"`s) will break things -- I can fix it if you don't have time though.

**Comment by auzelac91** [comment]
I think it's fixed now :)

**Comment by jgmize** [comment]
Thanks @auzelac91

**Pull request merged**